__tests__/users.ts

```diff
@@ -2252,3 +2294,189 @@ describe('mutation unsubscribePersonalizedDigest', () => {
     expect(res.errors).toBeFalsy();
   });
 });
+
+describe('mutation acceptFeatureInvite', () => {
+  const MUTATION = `mutation AcceptFeatureInvite($token: String!, $referrerId: ID!, $feature: String!) {
+    acceptFeatureInvite(token: $token, referrerId: $referrerId, feature: $feature) {
+      _
+    }
+  }`;
+
+  beforeEach(async () => {
+    await con.getRepository(Invite).save({
+      userId: '2',
+      campaign: CampaignType.Search,
+      limit: 5,
+      count: 1,
+      token: 'foo',
+    });
+  });
+
+  it('should require authentication', async () => {
+    await testQueryErrorCode(
+      client,
+      {
+        query: MUTATION,
+        variables: {
+          token: 'foo',
+          referrerId: 2,
+          feature: CampaignType.Search,
+        },
+      },
+      'UNAUTHENTICATED',
+    );
+  });
+
+  it('should return a 404 if the token and referrer mismatch', async () => {
+    loggedUser = '1';
+
+    await testQueryErrorCode(
+      client,
+      {
+        query: MUTATION,
+        variables: {
+          token: 'foo',
+          referrerId: 1,
+          feature: CampaignType.Search,
+        },
+      },
+      'NOT_FOUND',
+    );
+  });
+
+  it('should raise a validation error if the invite limit has been reached', async () => {
+    loggedUser = '1';
+
+    await con
+      .getRepository(Invite)
+      .update({ token: 'foo' }, { limit: 5, count: 5 });
+
+    await testQueryError(
```

Comment:
I think you can use `testMutationErrorCode` here for simplicity.

Reply:
in this case I am intentionally using `testQueryError` because I want to validate the error message itself as well, apart from the code